MathSearch.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+# wfLoadExtension( 'MathSearch' );
+
+# The link to your (local) MathWeb Search installation e.g. default 8080
+$wgMWSUrl = "http://localhost:8080";
+# The link to your (local) BaseX installation e.g. ntcir-11 set starting port 1984
+$wgMathSearchBaseX = "http://localhost:1984";
+$wgMathDebug = true;
+
+# this probably internal basex port
+# $wgMathSearchBaseXBackendUrl="http://localhost:1984/";
+
+$wgMathSearchBaseXBackendUrl="http://192.168.178.21:10043/";
```

Comment:
again use the container name.

Reply:
basex-rest (over restd) is not running in a container, but in external ide for debugging.